tests/testthat/output/trunc_mat/all-1-30-2.txt

```diff
@@ -1,7 +1,6 @@
-       a     b     c     d
-   <dbl> <int> <lgl> <chr>
-1      1     1  TRUE     a
-..   ...   ...   ...   ...
-Variables not shown: e
-  <fctr>, f <date>, and 2
-  more <...>.
+      a     b     c     d
+  <dbl> <int> <lgl> <chr>
+1     1     1  TRUE     a
+... with 1 more rows, and 4
```

Comment:
Do we need a test case where the length of a variable name is greater than the width?